src/modules/sensors/vehicle_magnetometer/VehicleMagnetometer.cpp

```diff
@@ -329,7 +329,7 @@ void VehicleMagnetometer::UpdateMagCalibration()
 
 					if (_calibration[mag_index].set_offset(mag_cal_offset)) {
 
-						PX4_INFO("%d (%" PRIu32 ") EST:%d offset: [%.2f, %.2f, %.2f]->[%.2f, %.2f, %.2f] (full [%.3f, %.3f, %.3f])",
+						PX4_INFO("%d (%" PRIu32 ") EST:%d offset: [%.3f, %.3f, %.3f]->[%.3f, %.3f, %.3f] (full [%.3f, %.3f, %.3f])",
```

Comment:
Some of these strings weren't being fully captured, maybe check a quick example.

Reply:
Seems to work
![mag_offset_commit](https://github.com/PX4/PX4-Autopilot/assets/14822839/94c98aea-7403-4881-b002-985d4611c0e2)